.github/workflows/ci.yaml

```
@@ -89,7 +89,7 @@ jobs:
             name: ci-artifacts
             path: artifacts-native${{ matrix.java }}.zip
   build-code-start:
-    name: Code Quarkus build - released Quarkus
+    name: Code Quarkus build
```

```suggestion
    name: Linux - Code Quarkus build
```
Again, consistency, but I'm not sure about this one, as it makes the name longer.